skiko/src/jsMain/kotlin/org/jetbrains/skia/Actuals.js.kt

```diff
@@ -8,18 +8,31 @@ actual fun <R> commonSynchronized(lock: Any, block: () -> R) {
 
 actual fun String.intCodePoints(): IntArray = TODO()
 
-actual class Pattern {
-    actual fun split(input: CharSequence): Array<String?>? = TODO()
-    actual fun matcher(input: CharSequence): Matcher = TODO()
+actual class Pattern constructor(regex: String) {
+    private val _regex = Regex(regex)
+
+    actual fun split(input: CharSequence): Array<String?>? = _regex.split(input).toTypedArray()
```

Comment:
Can't we return `Array<String>`, without nulls?

Reply:
Seems to be better indeed.

Reply:
agree. fixed